### PR TITLE
Update goods api to show version 2

### LINKED ISSUE #14: Update the "Digital Goods API" documentation for version 2
https://github.com/satoshipay/user-stories/issues/62

### PR DESCRIPTION
**Pull request by alinnow**
Fix #14

### CODE REVIEW

File: source/includes/_api.md
Comment (alinnow) [inline source/includes/_api.md]: In case anyone is curious: https://en.wikipedia.org/wiki/Dash#Em_dash

**Comment by alinnow** [comment]
If anyone tries to locally check it out and has problems with long/infinite page loads: https://github.com/lord/slate/issues/866#issuecomment-342835606

File: source/includes/_api.md
Comment (andywer) [reply source/includes/_api.md]: Fair enough, but I think there should be a space before and after the dash, right?

**Review by andywer — DISMISSED** [review]
I think there should be a space before and after the dash (top description).

Still approving, since it's a tiny thing.

File: source/includes/_api.md
Comment (alinnow) [reply source/includes/_api.md]: Debatable. Some style guides say none, some say hair spaces. I can make it hair spaces if you'd prefer

Comment (andywer) [reply source/includes/_api.md]: You are right and I really didn't see it coming. Looks wrong to me to not have spaces around, but look what Google even says:

bildschirmfoto 2017-12-04 um 09 49 34

**Comment by alinnow** [comment]
Merging this will auto-deploy to production. Wait for @TorstenStueber to merge.

**Review by levino — DISMISSED** [review]
Please deploy this to https://satoshipay.io/api2. You can either make a  new branch next to the master branch called eg: `version2` or use this feature branch for the source of the deployment. The idea is that anyone can see the final result before merging it over. We should only merge this in on december 7th, I think. Pinging @TorstenStueber on this too.

File: source/includes/_api.md
Comment (TorstenStueber) [inline source/includes/_api.md]: Seems like that link to `#html-endpoints` is broken.

File: source/includes/_api.md
Comment (TorstenStueber) [inline source/includes/_api.md]: The `c` of `shellc` should be part of `curl` in the next line.

**Review by TorstenStueber — DISMISSED** [review]
See comments.

**Comment by levino** [comment]
Will this be deployed to a staging url or not?

**Comment by alinnow** [comment]
@TorstenStueber said "no" before. I thought he would have commented as well.

**Comment by levino** [comment]
Okay, I am out of this PR then.

**Review by andywer — APPROVED** [review]

**Comment by andywer** [comment]
Looks good to me.

**Comment by andywer** [comment]
**DO NOT MERGE BEFORE DECEMBER 7TH**

(I almost did. This needs to be mentioned here in a prominent way.)

**Pull request merged**
